tests/netutils/main.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+￼* @author      Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
I think I can be removed from that. Nothing in this file is particularly me. ;-).